### PR TITLE
tests/main/interfaces-ssh-keys: tweak checks for openSUSE Tumbleweed

### PR DESCRIPTION
**Pull request by bboozzoo**
Since the 8.4p1 version of the openssh package, the vendor config files were
moved to /usr/etc/ssh.

### CODE REVIEW

**Review by mardy — APPROVED** [review]
LGTM, but I wonder if we can move the condition from being distro-specific to being dependent on the openssh version?

File: tests/main/interfaces-ssh-keys/task.yaml [hunk 1]
```diff
@@ -36,7 +36,14 @@ execute: |
     echo "And the snap is able to read public/private keys and ssh configuration files as well"
     test-snapd-sh.with-ssh-keys-plug -c "cat $TESTKEY"
     test-snapd-sh.with-ssh-keys-plug -c "cat $TESTKEY.pub"
-    test-snapd-sh.with-ssh-keys-plug -c "cat /etc/ssh/ssh_config"
+    if ! os.query is-opensuse-tumbleweed; then
+        test-snapd-sh.with-ssh-keys-plug -c "cat /etc/ssh/ssh_config"
+    else
+        # Tumbleweed: since updating to openssh 8.4p1, the vendor config files
+        # are at /usr/etc/ssh which is not accessible to snaps;
```
Comment (mvo5) [inline tests/main/interfaces-ssh-keys/task.yaml:43]: Does this mean we need to expand the interfaces-ssh-keys interface to include this new location?

Comment (bboozzoo) [reply tests/main/interfaces-ssh-keys/task.yaml:43]: I'm not quite sure what we can do in that interface now, we probably should discuss. There is no /usr/etc in the core* bases, so nothing we can mount on. We can create a temporary /etc/ssh and mount /usr/etc/ssh on top of this, which could work? provided that the host config does not reference any paths that are outside. Also, note that AFAICT openssh in opensuse is being patched to support that non-standard location.

Comment (mvo5) [reply tests/main/interfaces-ssh-keys/task.yaml:43]: Yeah, let's ignore for now until we have a use-case. AIUI the /etc/ssh is still there on suse and still the way to do admin changes so it should be good.

**Review by mvo5 — APPROVED** [review]

**Pull request merged**